tests/cpp/test_index_ops.cpp

```diff
@@ -0,0 +1,90 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <csrc/exceptions.h>
+#include <gtest/gtest.h>
+
+#include <ops/all_ops.h>
+#include <runtime/fusion_executor_cache.h>
+#include <tests/cpp/utils.h>
+#include <tests/cpp/validator.h>
+
+namespace nvfuser::IndexPutAccumulate {
```

Comment:
nit. No need to create a new named namespace.

```suggestion
namespace nvfuser {
namespace {
```

Reply:
I was thinking about adding a more general index_ops, hence the namespace.

But yeah we can certainly keep just a single file for the operation.